bin/install-gutenboarding-design-thumbnails-dependencies.js

```diff
@@ -1,48 +0,0 @@
-#!/usr/bin/env node
-
-/*
- * This file installs dependencies we require to generate the gutenboarding design thumbnails.
- * It is not used for any other purpose.
- * See: bin/generate-gutenboarding-design-thumbnails.js
- * @Automattic/create
- */
-const spawnSync = require( 'child_process' ).spawnSync;
-const fs = require( 'fs' );
-
-// run a distclean to clean things up. just ci is not enough with the monorepo.
-const cleanResult = spawnSync( 'yarn', [ 'run', 'distclean' ], {
-	shell: true,
-	stdio: 'inherit',
-} );
-if ( cleanResult.status ) {
-	console.error( 'failed to clean: exited with code %d', cleanResult.status );
-	process.exit( cleanResult.status );
-}
-
-const installResult = spawnSync( 'yarn', [ 'install', '--frozen-lockfile' ], {
-	shell: true,
-	stdio: 'inherit',
-	env: { ...process.env },
-} );
-if ( installResult.status ) {
-	console.error( 'failed to install: exited with code %d', installResult.status );
-	process.exit( installResult.status );
-}
-
-/*
- * Add screenshot dependencies
- * I didn't want to add these to dependencies just for this script since these are kinda heavy ones)
- * https://www.npmjs.com/package/capture-website
- * https://www.npmjs.com/package/sharp
- */
-const installDependencies = spawnSync( 'yarn', [ 'add', '-W', 'capture-website', 'sharp' ], {
```

Comment:
While we're here, I think we can remove the `capture-website` dependency, too. This seems to be its sole usage.

Reply:
done!